components/screens/ShoppingCart.js

```diff
@@ -1,13 +1,63 @@
-import React from 'react';
-import { View, ScrollView, Text, StyleSheet } from 'react-native';
+import React, {useState} from 'react';
+import { Alert, Button } from 'react-native';
+import { View, ScrollView, Text, StyleSheet, Image } from 'react-native';
 import NavBar from '../NavBar';
 
-function ShoppingCart({navigation}) {
+function ShoppingCart({ screenProps,navigation }) {
+
+    const [shoppingCart, setShoppingCart] = useState(screenProps.shoppingCart); 
+
+    let totalPrice = 0;
+
+    const createList = () => {
+        return shoppingCart.map((element, index) => {
+            totalPrice += element.price;
+            return (
+                <View style={styles.product_container} key={index}>
+                    <Image style={styles.product_image} source={element.image} />
+                    <Text style={styles.product_name}>{element.name}</Text>
+                    <Text style={styles.product_price}>{element.price}</Text>
```

Comment:
Instead of doing `styles.product_image`. You can destructure above and use just `product_price` here 
easy to read code.